app/services/discovery_report.rb

```diff
@@ -8,8 +8,7 @@ class DiscoveryReport
   attr_reader :batch, :summary
   attr_accessor :error_message, :objects_had_errors
 
-  delegate :staging_location, :content_md_creation, :manifest, :project_style, :using_file_manifest, to: :batch
```

Comment:
is this the only external usage of "using_file_manifest" in batch?  If so ... can we lose that exposure?  Also, why isn't it named with a question mark in the batch class?

Reply:
I don't know.